### PR TITLE
meta/redis: support new backup

### PR DESCRIPTION
**Pull request by jiefenghuang**

### CODE REVIEW

File: pkg/meta/redis_bak.go [hunk 1]
```diff
@@ -19,16 +19,922 @@
...
+	if len(mbs) == 0 {
+		return nil
+	}
+	return m.rdb.ZAdd(ctx, m.delfiles(), mbs...).Err()
+}
+
+func (m *redisMeta) loadSliceRefs(ctx Context, msg proto.Message) error {
+	slices := make(map[string]interface{})
+	for _, p := range msg.(*pb.Batch).SliceRefs {
+		slices[m.sliceKey(p.Id, p.Size)] = strconv.Itoa(int(p.Refs - 1))
```
Comment (davies) [inline pkg/meta/redis_bak.go:826]: small batch

File: pkg/meta/redis_bak.go [hunk 2]
```diff
@@ -19,16 +19,922 @@
...
+		}
+	}
+	return execPipe(ctx, pipe)
+}
+
+func (m *redisMeta) loadDelFiles(ctx Context, msg proto.Message) error {
+	batch := msg.(*pb.Batch)
+	mbs := make([]redis.Z, 0, len(batch.Delfiles))
+	for _, pd := range batch.Delfiles {
+		mbs = append(mbs, redis.Z{
```
Comment (davies) [inline pkg/meta/redis_bak.go:812]: small batch

File: pkg/meta/redis_bak.go [hunk 3]
```diff
@@ -19,16 +19,922 @@
...
+		} else {
+			cs[m.counterKey(strings.ToLower(c.Key))] = c.Value
+		}
+	}
+	return m.rdb.MSet(ctx, cs).Err()
+}
+
+func (m *redisMeta) loadNodes(ctx Context, msg proto.Message) error {
+	batch := msg.(*pb.Batch)
+	nodes := make(map[string]interface{}, redisBatchSize)
```
Comment (davies) [inline pkg/meta/redis_bak.go:709]: why not pipeline?

Comment (jiefenghuang) [reply pkg/meta/redis_bak.go]: MGET is faster than pipeline.

**Pull request merged**